pkg/cli/admin/inspect/resource.go

```diff
@@ -126,8 +211,10 @@ func gatherRelatedObjects(context *resourceContext, unstr *unstructured.Unstruct
 		return err
 	}
 
-	relatedObjReferences = append(relatedObjReferences, obtainWebhookService(unstr)...)
```

Comment:
I could move some of the blocks from above to here.  it seemed a little more straightforward to add it to the main flow.